### PR TITLE
XS✔ ◾ Geordie88 patch 2

### PR DESCRIPTION
**Pull request by Geordie88**
## Reason for change (Issue, Email, conversation + reason, etc)

{{ REASON FOR CHANGE - WHAT WAS THE TRIGGER FOR THIS CHANGE }}
I got a email from Uly asking me to make a new category with 10 rules ( he did all the markdown)
{{ SUMMARY OF CHANGES }}
added a category to rules to management and added 10 rules

### CODE REVIEW

File: rules/favicon/favicon-good.jpg
Comment (bradystroud) [inline rules/favicon/favicon-good.jpg:1]: Why are we using such old screenshots here?

**Review by bradystroud — APPROVED** [review]
Approving the Markdown sytax

The content was written by Uly and checked by Gert

File: categories/management/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ index:
   - rules-to-better-product-owners
   - rules-to-better-software-consultants-dealing-with-clients
   - rules-to-better-software-consultants-working-in-a-team
+  - rules-to-betters-scaled-agile-framework-(SAFe)
```
Comment (jaydenalchin) [inline categories/management/index.md:13]:
```suggestion
  - rules-to-better-scaled-agile-framework-(SAFe)
```

File: categories/management/rules-to-betters-scaled-agile-framework-(SAFe).md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+type: category
+title: Rules to Better Scaled Agile Framework (SAFe)
+guid: fe191cd3-36bc-4a61-900a-973780bb1cd3
+uri: rules-to-betters-scaled-agile-framework-(SAFe)
```
Comment (jaydenalchin) [inline categories/management/rules-to-betters-scaled-agile-framework-(SAFe).md:5]:
```suggestion
uri: rules-to-better-scaled-agile-framework-(SAFe)
```

Comment (jaydenalchin) [reply categories/management/rules-to-betters-scaled-agile-framework-(SAFe).md:5]: Typo - Note you'll need to fix it in the category file name as well

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

**Review by Geordie88 — COMMENTED** [review]
fixing title

**Review by jaydenalchin — APPROVED** [review]

**Review by UlyssesMaclaren — APPROVED** [review]

**Pull request merged**